packages/contracts-bedrock/test/properties/medusa/Protocol.properties.t.sol

```diff
@@ -3,13 +3,30 @@ pragma solidity ^0.8.25;
 
 import { TestBase } from "forge-std/Base.sol";
 
+import { ITokenMock } from "@crytic/properties/contracts/ERC20/external/util/ITokenMock.sol";
 import { EnumerableMap } from "@openzeppelin/contracts/utils/structs/EnumerableMap.sol";
+import { CryticERC20ExternalBasicProperties } from
+    "@crytic/properties/contracts/ERC20/external/properties/ERC20ExternalBasicProperties.sol";
 import { ProtocolHandler } from "./handlers/Protocol.handler.t.sol";
 import { OptimismSuperchainERC20 } from "src/L2/OptimismSuperchainERC20.sol";
 
-contract ProtocolProperties is ProtocolHandler {
+contract ProtocolProperties is ProtocolHandler, CryticERC20ExternalBasicProperties {
```

Comment:
shouldn't we include the other ones (mintable, burnable, etc)?

Reply:
we could do that, but not in the context of fuzzing the entire protocol, since burning and minting have rules around them and can't be called willy-nilly by the fuzzer

I tried having a separate `OptimismSuperchainERC20` contract and running ToB invariants (including mintable/burnable) on it, it was doable but I had to override `mint` and `burn` so at that point it was only testing solady's ERC20 implementation. 
If you deem that valuable enough though, I could add it.